header.php

```diff
@@ -26,9 +26,9 @@
 
 <body <?php body_class( 'site-wrapper' ); ?>>
 
-	<?php wp_body_open(); ?>
+	<?php wd_s_wp_body_open(); ?>
```

Comment:
Noting that we should keep `wp_body_open()` and instead remove the (weirdly named) `wd_s_wp_body_open()` as this theme already specifies a minimum WP version of 5.6 so the backwards compatibility is unnecessary. See https://generatewp.com/wordpress-5-2-action-that-every-theme-should-use/

Reply:
Sounds good to me.